PCbuild/pcbuild.proj

```diff
@@ -77,7 +77,7 @@
     <ExtensionModules Include="@(ExternalModules->'%(Identity)')" Condition="$(IncludeExternals)" />
     <Projects Include="@(ExtensionModules->'%(Identity).vcxproj')" Condition="$(IncludeExtensions)" />
     <!-- Test modules -->
-    <TestModules Include="_ctypes_test;_testbuffer;_testcapi;_testinternalcapi;_testembed;_testimportmultiple;_testmultiphase;_testsinglephase;_testconsole" />
+    <TestModules Include="_ctypes_test;_testbuffer;_testcapi;_testinternalcapi;_testembed;_testimportmultiple;_testmultiphase;_testsinglephase;_testconsole;_testclinic" />
```

Comment:
I believe there should be similar places to add the new module under `PC/layout` and `Tools/msi/test`, otherwise it won't get into the installers.

Reply:
Seems `PC/layout` is handled by a glob:

https://github.com/python/cpython/blob/41de54378d54f7ffc38f07db4219e80f48c4249e/PC/layout/main.py#L33

For `Tools/msi/test` I'm kinda lost; I'm not sure what to do with those files :(

Reply:
https://github.com/python/cpython/blob/main/Tools/msi/test/test_files.wxs#L3

Similar to pcbuild.proj, there's a macro that lists the names to generate. Just needs to be added there

Reply:
Tools/msi/test/test\_files updated in a3e7085f70a3e588d5f1b3fbb66f95a098bbdb3d